app/views/issue_templates/new.html.erb

```diff
@@ -15,7 +15,7 @@
                             url: { controller: 'issue_templates', action: 'new', project_id: @project },
                             html: { id: 'issue_template-form', class: nil, multipart: false } do |f| %>
 
-          <%= render partial: 'form', locals: { f: f } %>
+          <%= render partial: 'form', locals: { f: f, checklist_enabled: checklist_enabled } %>

```

Comment:
__[rails_best_practices]__
* simplify render in views [:ref](http://rails-bestpractices.com/posts/2010/12/04/simplify-render-in-views/)

<!-- Comment by [SideCI](https://sideci.com) -->